core/src/main/java/org/frankframework/task/TimeoutGuard.java

```diff
@@ -51,14 +51,11 @@ public Killer() {
 
 		@Override
 		public void run() {
-			log.warn("Thread [{}] executing task [{}] exceeds timeout of [{}] s, interrupting",
-				timeoutThread.getName(), description, timeout);
+			log.warn("Thread [{}] executing task [{}] exceeds timeout of [{}] s, interrupting. Created from source point:",
+				timeoutThread.getName(), description, timeout, source);
```

Comment:
@nielsm5 Hier heb ik iets gesleuteld om te kunnen debuggen waarom een test fout ging -- er werd een TimeoutGuard getriggered waarvan we niet wisten waar die vandaan kwam.

Met deze stacktrace konden we dat wel zien.

Willen we deze extra log er in laten zitten bij iedere warning van een TimeoutGuard? Of willen we dat er conditioneel in hebben / alleen bij loglevel DEBUG?

Reply:
Ik had een stacktrace toegevoegd aan de `warn` log.

Ik heb die nu verplaatst naar `debug` level en de oorspronkelijke `warn` log hersteld.